src/internal/schema.ts

```diff
@@ -4,17 +4,42 @@ import { IModelOptions } from '../typegoose';
 import { EmptyVoidFn, NoParamConstructor } from '../types';
 import { DecoratorKeys } from './constants';
 import { buildSchemas, hooks, plugins, schemas, virtuals } from './data';
+import { getParentClasses } from './utils';
 
 /**
  * Private schema builder out of class props
```

Comment:
Could you make tsdoc for the new _buildSchema & getSchemaForClass differ from each other (to better distinguish them)

Reply:
fixed in e3bea3322cb7ae97825578667a1b4f39eb12c492